README.md

```diff
@@ -106,3 +133,6 @@ The manual process of creating a new version is to:
        ]
     }
     ```
+
```

Comment:
some not needed empty spaces

Reply:
Hi. I have worked on the changes suggested. And thanks for the reference Doc. It helped. Please review.